src/WebJobs.Script/Workers/Rpc/MessageExtensions/RpcMessageExtensionUtilities.cs

```diff
@@ -77,18 +80,21 @@ public static object ConvertFromHttpMessageToExpando(RpcHttp inputMessage)
             return new Tuple<string, string, CookieOptions>(cookie.Name, cookie.Value, cookieOptions);
         }
 
-        private static SameSiteMode RpcSameSiteEnumConverter(RpcHttpCookie.Types.SameSite sameSite)
+        private static bool TryGetRpcSameSiteEnumConverter(RpcHttpCookie.Types.SameSite sameSite, out SameSiteMode sameSiteMode)
         {
+            sameSiteMode = SameSiteMode.None;
             switch (sameSite)
             {
                 case RpcHttpCookie.Types.SameSite.Strict:
-                    return SameSiteMode.Strict;
+                    sameSiteMode = SameSiteMode.Strict;
+                    return true;
                 case RpcHttpCookie.Types.SameSite.Lax:
-                    return SameSiteMode.Lax;
+                    sameSiteMode = SameSiteMode.Lax;
+                    return true;
                 case RpcHttpCookie.Types.SameSite.None:
-                    return SameSiteMode.None;
+                    return false;
```

Comment:
If we get the protobuf message default "RpcHttpCookie.Types.SameSite.None" value, do not add a SameSite property. In V2, SameSiteMode.None (the default) becoming  would become an cookie with no samesite property set. In .NET Core 3.1, SameSiteMode.None becomes an additional property on the cookie `samesite=none`. This change preserves V2 behavior. 

The addition of `samesite=none` introduced in .NET Core 3.1 will be addressed here: https://github.com/Azure/azure-functions-nodejs-worker/pull/270

We do not have to worry about the default when cookieOptions.SameSite is not present because this is not a valid case for the cookies we build.